rules/docker-gnmi.mk

```diff
@@ -34,5 +34,10 @@ $(DOCKER_GNMI)_RUN_OPT += -t
 $(DOCKER_GNMI)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 $(DOCKER_GNMI)_RUN_OPT += -v /etc/localtime:/etc/localtime:ro
 $(DOCKER_GNMI)_RUN_OPT += -v /var/run/dbus:/var/run/dbus:rw
+$(DOCKER_GNMI)_RUN_OPT += -v /:/mnt/host:ro
```

Comment:
Can you add some comments here to specify purpose of requiring mount of individual paths here?
1. / - To check disk space and monitor usage
2. /tmp - To download the image binary and other upgrade package.
3. ....

Reply:
Why is `/var/tmp/` needed to be mounted?